src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -395,34 +400,38 @@ private function entered(object $subject, ?Transition $transition, Marking $mark
             foreach ($transition->getTos() as $place) {
                 $this->dispatcher->dispatch($event, sprintf('workflow.%s.entered.%s', $this->name, $place));
             }
+        } elseif (!empty($this->definition->getInitialPlaces())) {
```

Comment:
This is not related to the current PR. Could you revert it? thanks

Reply:
without that, the DEFAULT_INITIAL_CONTEXT makes no sense.

it's a little far fetched, but I believe adding this context enhancement is a good moment to add this default context

Unless you don't want a DEFAULT_INITIAL_CONTEXT ?